### PR TITLE
Feature/apvs 211/record manually processed payments

### PR DESCRIPTION
**Pull request by pwright08**
apvs-211 - record manually processed payments
 
- updated `get-claims-pending-payment` to record the total payment amount including manually processed expenses, and then remove the manually processed amount from the amount to be paid
- changed payment query to pay for claim expenses with status `APPROVED-DIFF-AMOUNT` as well as `APPROVED`

## Checklist

- [ ] Unit tests
- [ ] Code coverage checked
- [ ] Coding standards
- [ ] Error Handling
- [ ] Security/performance considered?
- [ ] Deployment changes considered?
- [ ] README updated

### CODE REVIEW

File: app/services/data/get-claims-pending-payment.js [hunk 1]
```diff
@@ -21,17 +26,51 @@ module.exports = function () {
...
     .whereIn('IntSchema.Claim.Status', [claimStatuses.APPROVED, claimStatuses.AUTOAPPROVED])
-    .where({'IntSchema.ClaimExpense.Status': claimStatuses.APPROVED})
+    .whereIn('IntSchema.ClaimExpense.Status', [claimExpenseStatuses.APPROVED, claimExpenseStatuses.APPROVED_DIFF_AMOUNT, claimExpenseStatuses.MANUALLY_PROCESSED])
+    .andWhere(function () {
+      this.where('IntSchema.ClaimDeduction.IsEnabled', true)
+      .orWhereNull('IntSchema.ClaimDeduction.ClaimDeductionId')
+    })
     .whereNull('IntSchema.Claim.PaymentStatus')
     .groupBy(selectColumns)
     .then(function (results) {
```
Comment (gdouglas1) [inline app/services/data/get-claims-pending-payment.js]: Split into a different function to make it more readable

File: app/services/data/get-claims-pending-payment.js [hunk 2]
```diff
@@ -21,17 +26,51 @@ module.exports = function () {
...
+      results.forEach(function (result) { return claimIds.push(result.ClaimId) })
+      claimResults = results
+      return knex('IntSchema.ClaimExpense')
+        .sum('ApprovedCost as ManuallyProcessedCost')
+        .select('ClaimId')
+        .groupBy('ClaimId')
+        .where('Status', 'MANUALLY-PROCESSED')
+        .whereIn('ClaimId', claimIds)
+    })
+    .then(function (manuallyProcessedExpenseCostsPerClaim) {
```
Comment (gdouglas1) [inline app/services/data/get-claims-pending-payment.js]: Same as above

**Comment by stevenalexander** [comment]
Closing until corrected

**Comment by pwright08** [comment]
Fixed database updates to use 2 decimal places, still need to set the amount written to csv to 2 decimal places also

**Pull request merged**